go.mod

```diff
@@ -1,6 +1,6 @@
 module helm.sh/helm/v3
 
-go 1.17
+go 1.18
```

Comment:
We should (in a separate PR; this one is a bugfix for the previous regression) bump to 1.19. As now golang 1.20 is released, that obsoletes 1.18.

Reply:
@gjenkins8 Should I create a new PR with go **1.19/1.20** changes or close this PR?

Reply:
I think another PR (please). To separate the concerns of bugfix vs feature or maintenance work.

Reply:
@gjenkins8 Closing this PR as I will create a new PR with the requested go.mod version!

Reply:
Here's the link for new PR: https://github.com/helm/helm/pull/11847